dataprofiler/data_readers/filepath_or_buffer.py

```diff
@@ -0,0 +1,31 @@
+from io import open, StringIO, BytesIO
+
+from . import data_utils
+
+class FileOrBufferHandler():
+    def __init__(self, stream_or_buffer, open_method='r', options=dict()):
```

Comment:
instead of having options, lets just make seek an arg here, e.g. 
`def __init__(self, stream_or_buffer, open_method='r', seek=None):`